android/app/src/main/java/com/bounswe2020group3/paperlayer/register/RegisterPresenter.kt

```diff
@@ -1,21 +1,52 @@
 package com.bounswe2020group3.paperlayer.register
 
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import com.bounswe2020group3.paperlayer.R
+import com.bounswe2020group3.paperlayer.profile.ProfileContract
+import com.bounswe2020group3.paperlayer.profile.ProfileModel
 import com.bounswe2020group3.paperlayer.register.RegisterContract
+import com.bounswe2020group3.paperlayer.register.data.Register
+import io.reactivex.disposables.CompositeDisposable
+import kotlinx.coroutines.flow.callbackFlow
+import retrofit2.Callback
+import retrofit2.Call
+import retrofit2.Response
 import timber.log.Timber
 
 class RegisterPresenter(view: RegisterContract.View)
     : RegisterContract.Presenter {
     private var view: RegisterContract.View? = view
-
+    private var model: RegisterContract.Model = RegisterModel()
+    private var disposable = CompositeDisposable()
```

Comment:
You should clear this disposable on destroy

Reply:
Ok, thank you I corrected that mistake